src/services/websockets.js

```diff
@@ -20,26 +20,32 @@ export function initialise (actionCreatorBinder) {
   } = actionCreatorBinder({...SearchResultActions, ...TagActions});
   primus.on('data', function received (data) {
     console.log('incoming socket data', data);
-    if (data.connection) {
-      saveSocketConnectionId(data.connection);
+    if (data.graphql) {
+      saveSearchResult(data);
+    }
+  });
+
+  primus.once('open', (data) => {
```

Comment:
@lennym does this still support the scenario you described in #180 where your device looses Wifi/3G ? (*will it re-connect*)